src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -79,6 +85,15 @@ private static bool TryReadMemoryValueFromFile(string path, out ulong result)
                         result = checked(ulongValue * multiplier);
                         return true;
                     }
+
+                    string value = Encoding.UTF8.GetString(bytes);
+                    if ("max".Equals(value, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
I don't understand what this check is doing.  If Equals returns true, we set result to 0 and return false.  If Equals returns false, we'll fall through to the exist code path that sets result to 0 and returns false.  So... why do we need this extra check at all?

Reply:
Strictly speaking, we don't. I figured I would handle the entire set of valid input values here explicitly. `max` is the only non-numeric value possible and I wanted to handle it in the body along with all other numeric values.

Should I add a check for other values and warn if anything other than a numeric value and `max` is seen?